docs/guide/repository.md

```diff
@@ -734,8 +604,6 @@ Example usage of the cache service:
     return $myObject;
 ```
 
-For more info on usage, take a look at [Stash's documentation](http://stash.tedivm.com/).
```

Comment:
Instead point to Symfony doc, like:
`For more info on usage, take a look at [Symfony Cache's documentation](https://symfony.com/doc/3.4/components/cache.html).`


And example should be changed to:

```php
// Example
$cacheItem = $pool->getItem("myApp-object-${id}");
if ($cacheItem->isHit()) {
    return $cacheItem->get();
}

$myObject = $container->get('my_app.backend_service')->loadObject($id)
$cacheItem->set($myObject);
$cacheItem->tag(['myApp-category-' . $myObject->categoryId]);
$pool->save($cacheItem);

return $myObject;
```